gobblin-modules/gobblin-kafka-common/build.gradle

```diff
@@ -44,6 +46,9 @@ dependencies {
   testCompile project(":gobblin-test-utils")
   testCompile externalDependency.mockito
   testCompile externalDependency.testng
+
+  jmh 'org.openjdk.jmh:jmh-core:1.17.3'
```

Comment:
Now that we seem to have multiple modules using jmh, can you move out the version definition to common place like definitions.gradle and refer to it like we refer to them normally?